wscript

```diff
@@ -42,9 +43,36 @@ def options(opt):
     opt.add_option('--shared', action='store_true', help='build shared library', dest='build_shared')
     opt.add_option('--tests', action='store_true', help='compile tests or not', dest='tests')
     opt.add_option('--python', action='store_true', help='compile python bindings', dest='pybind')
+    opt.add_option('--utheque-only', action='store_true', help='only install the URDF library (utheque) / deactivate RobotDART', dest='utheque_only')
+    opt.add_option('--no-robot_dart', action='store_true', help='only install the URDF library (utheque) / deactivate RobotDART', dest='utheque_only')
 
 
 def configure(conf):
+    if not conf.options.utheque_only:
+        robot_dart_ok = True
```

Comment:
`robot_dart_ok` seems not used anywhere..

Reply:
Good catch. I moved to `env["...`